doc/JSON_INFO.md

```diff
@@ -1265,8 +1265,8 @@ See also VEHICLE_JSON.md
 "price": 100,                    // Used when bartering with NPCs. For stackable items (ammo, comestibles) this is the price for stack_size charges. Can use string "cent" "USD" or "kUSD".
 "price_post": "1 USD",           // Same as price but represent value post cataclysm. Can use string "cent" "USD" or "kUSD".
 "material": ["COTTON"],          // Material types, can be as many as you want.  See materials.json for possible options
-"cutting": 0,                    // (Optional, default = 0) Cutting damage caused by using it as a melee weapon
-"bashing": -5,                   // (Optional, default = 0) Bashing damage caused by using it as a melee weapon
+"cutting": 0,                    // (Optional, default = 0) Cutting damage caused by using it as a melee weapon.  Cannot be negative?
+"bashing": 0,                   // (Optional, default = 0) Bashing damage caused by using it as a melee weapon.  Cannot be negative?
```

Comment:
```suggestion
"bashing": 0,                   // (Optional, default = 0) Bashing damage caused by using it as a melee weapon.
```
As above.

Reply:
As above.  What ever we decide to do for cutting I'll do for bashing too.  So I'll mark this resolved, because resolving above will ensure this gets resolved.